Gemfile

```diff
@@ -48,16 +48,17 @@ group :development, :test do
 
   gem 'pry-rails'
   gem 'rb-readline'
+
+  gem 'rspec-rails', '~> 4.0.1'
 end
 
 group :test do
-  gem 'database_cleaner'
```

Comment:
"But how is the database cleaned up between tests?"

Rails handles that for us in its test suite via db checkpoints and transactions. It's handling of capybara also allows it to have a fresh db per test while cucumber needed DB cleaner due to the web server not talking to the actual test runner.